CONTRIBUTING.md

```diff
@@ -54,8 +54,17 @@
     `auto y = std::vector<size_t>{};`.
   * Namespaces: Do not create nested namespaces, do not import namespaces.
   * Consider structured bindings: `const auto& [iterator, added] = unordered_map.emplace(...);`
-  * Prefer braced control statements (`if (...) { x(); }` over `if (...) x();`). For multi-line statements, this is
-    mandatory.
+  * Use braced control statements (`if (...) { x(); }` instead of `if (...) x();`). This is even enforced for single
+    line blocks (via clang-format). The only exception are empty blocks (e.g., `while (!ready) {}`).
+  * Use braced control statements. This is even enforced for single line blocks (via clang-format). Unless the blocks
+    are empty (e.g., `while (!ready) {}`), add line breaks. Instead of `if (...) x();` (or `if (...) { x(); }`), write:
```

Comment:
I would focus on the essential information here and would remove the detail about clang-format.

```suggestion
  * Use braced control statements, even for single-line blocks. Unless the blocks
    are empty (e.g., `while (!ready) {}`), add line breaks. Instead of `if (...) x();` (or `if (...) { x(); }`), write:
```